### PR TITLE
Schedule page.

### PR DESCRIPTION
**Pull request by strand**

### CODE REVIEW

File: app/data/events.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
```
Comment (khjrtbrg) [inline app/data/events.yml]: :grey_question: It doesn't look like this is getting used anywhere? Can we delete it?

Comment (strand) [inline app/data/events.yml]: Yes. Good catch, I think I created `schedule.yml` instead of moving this file.

**Comment by khjrtbrg** [comment]
:package: http://osfeels-beta.herokuapp.com/schedule

**Comment by strand** [comment]
Some points of feedback:

- I *love* the hearts and mustard lines. This is a creative way to display the events in any given day. Did you consider any other icon/line color combos? I think the smiley and blue would be a reasonable alternative, and maybe we could alternate the lines on each day.
- I think we should use pink on the headers for organizer presented pieces of content. Blue and purple do not contrast enough to convey that the headers are semantically different.
- Unless there is specific behavior you are targeting to a particular device size, only use `col-med-*` attributes. `col-md-*` attributes make the page single page at iPad or smaller. See also: http://getbootstrap.com/css/#grid-options
- The menu should read "schedule", not "2015 schedule".
- the time column should take up less space. I think that `col-med-4` for the time column and `col-med-8` for the event column would be ideal.
- Holy crap! I love this page!

I hope this helps.

**Comment by strand** [comment]
Oh, one other request… please add anchors so we can use fragment identifiers to link to specific parts of the day. The simplest version of this would link to `friday-morning`, `friday-afternoon`, `saturday-morning`,  and `saturday-afternoon`.

**Comment by khjrtbrg** [comment]
@strand :purple_heart: Thanks for the feedback, I'll get it updated tomorrow!

**Comment by khjrtbrg** [comment]
:wave: @strand:
- Up-to-date version is on [beta](http://osfeels-beta.herokuapp.com/schedule)
- I originally added "2015" to the link for the schedule since I thought "schedule" blended in too much with the links for speakers and sponsors. I reverted it, but maybe the link order should get mixed up a little?
- I added the anchors, but they're kind of gnarly. Let me know if you have a better idea for how to do it!

**Comment by strand** [comment]
There will be tweaks, but this should be good to go. Oh no conflicts!

**Pull request merged**